src/components/collapsible.tsx

```diff
@@ -25,16 +21,27 @@ interface Props {
    * If the element should get transparent on hover
    */
   opacityOnHover?: boolean
+  /**
+   * Forces the collapsible to collapse
+   *   this can be used to allow control of the collapsed state outside of the component
+   *   an example use-case is controlling the focus of children collapsibles by the parent
+   *   component to ensure that only one is open at the time
+   */
+  forceCollapse?: boolean
 }
 
 const Collapsible: FC<Props> = ({
   renderToggle,
-  isInitiallyCollapsed = true,
   children,
   onChange,
   opacityOnHover = true,
+  forceCollapse = true,
 }) => {
-  const [isCollapsed, setIsCollapsed] = useState(isInitiallyCollapsed)
+  const [isCollapsed, setIsCollapsed] = useState(forceCollapse)
+
+  useEffect(() => {
+    setIsCollapsed(forceCollapse)
+  }, [forceCollapse])
```

Comment:
I dont think that we need the useEffect here because we initiate the state with the prop forceCollapse and we never change it internally